lib/improver/cube_combiner.py

```diff
@@ -152,6 +198,14 @@ def process(self, cube_list, new_diagnostic_name,
         data_type = cube_list[0].dtype
         result = cube_list[0].copy()
 
+        # If cube has coord bounds that we want to expand
+        if expanded_coord:
```

Comment:
Is there any reason to have this option here, rather than after calling `self.combine`, as this operation is conceptually similar to the amend_metadata functionality?